src/patternfly/_variables.scss

```diff
@@ -149,4 +149,15 @@
   // A11y
   --pf-global--target-size--MinWidth: #{$pf-global--target-size--MinWidth};
   --pf-global--target-size--MinHeight: #{$pf-global--target-size--MinHeight};
+
+  // Images
+  --pf-c-background-image--BackgroundImage--xs: #{$pf-global--pf-c-background-image--BackgroundImage--xs};
```

Comment:
These are not global variables. They belong to the component file.

Reply:
When they are located in the component file, they cannot be overridden.

Reply:
To override we would need to target the pf-c-background-image class. It is similar to the other overrides that were made here: https://github.com/project-koku/koku-ui/blob/master/src/pages/dashboard/dashboard.styles.ts#L35 This was a limitation/need brought on by the fact that variables are added inside the class. This was needed since updating them was causing problems with the root variables. Andres can speak more on to that. He had a pretty good demo of it. Any of these should be able to be overridden by doing something like
```css
body .pf-c-background-image { ...override here }
```
That should increase the specitivity (speeling?) enough.

See comment here: https://github.com/patternfly/patternfly-next/pull/310#discussion_r194050069

Reply:
This screenshot shows what happens when we add variables to the component CSS, then embed that same CSS in our react-core components. My styles are defined first, but the component is output last, overriding my variables.

![screen shot 2018-10-02 at 3 39 44 pm](https://user-images.githubusercontent.com/17481322/46372547-7610c680-c659-11e8-9aba-596e4484e569.png)

Reply:
This is a mixture of a few things
1. Injection order as you stated. This can be avoided by making the selector `body .pf-c-background-image` instead. Adding the body reference should make it more specific and force it to use it instead.
2. Webpack css loaders will likely still try to load all the image paths even if they are overridden by a css variable. This can be avoided by not using a css loader for the background-image css. This is somewhat unavoidable in this repo (react shouldn't run into this). Since this reop is using SASS you might be better off doing the override here with SASS, and the override with the custom properties in the react repo.

Note. For the react repo instead of using import foo from 'bar.css' for the styles you are specifying for overrides use `@patternfly/react-styles` you can either use the `injectGlobal` or use `StyleSheet.create`. `StyleSheet.create` will be preferable since you can scope it to the div itself.
```jsx
const styles = StyleSheet.create({
  backgroundExample: `
    & .pf-c-background-image { //all overrides in here }
 `
});

class Example extends React.Component {
  render()  {
    return (
      <div className={css(styles.backgroundExample)}>
        <BackgroundImage />
      </div>
    )
  }
}